ledger-api/3rdparty/grpc-definitions/google/rpc/status.proto

```diff
@@ -1,6 +1,3 @@
-// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-// SPDX-License-Identifier: Apache-2.0
-
```

Comment:
It looks like currently we would have to keep this to make `dade-copyright-headers` happy. @moritzkiefer-da @garyverhaegen-da Is there another option and, if not, do you see a problem with that?

Reply:
You can place a `NO_AUTO_COPYRIGHT` file in the directory to disable it. That said, I’m not sure why you need to copy this file in the first place. We already have the same protobuf as a Bazel dep via `@com_github_googleapis_googleapis//google/rpc:status_proto`. Why do we need it twice?

Reply:
I'll try and look into how to use the one from the Bazel dep with `buf`.

Reply:
It looks like bazel deps are cached at the user level but `buf` forbids access to files outside of the configuration root (in the work tree). I think the script would have to copy the file into the work tree. I wouldn't like the check script to know individual protos, so perhaps the `NO_AUTO_COPYRIGHT` solution is better in this case. wdyt?

Reply:
hm that’s a bit sad. I don’t like having to keep the two in sync but I guess status.proto is unlikely to change frequently if at all so NO_AUTO_COPYRIGHT is fine with me.

Reply:
Moved it to the already existing `/3rdparty`. Thanks!